metomi/rose/opt_parse.py

```diff
@@ -215,7 +327,10 @@ class RoseOptionParser(OptionParser):
                 "action": "store",
                 "dest": "diff_tool",
                 "default": None,
-                "help": "Specify an alternate diff tool like diffuse.",
+                "help": (
+                    "Specify an alternate diff tool."
+                    "\nE.G: diffuse, vimdiff or kompare."
```

Comment:
If you wrote "Exempli gratia" in full you'd have

```suggestion
                    "\nE.g: diffuse, vimdiff or kompare."
```
or you could just skip the Latin?
```suggestion
                    "\nFor example: diffuse, vimdiff or kompare."
```

Reply:
Capitalisation of e.g is mixed in all repos, not planning on addressing that right now.